Library/Formula/luvi.rb

```diff
@@ -0,0 +1,19 @@
+class Luvi < Formula
+  desc "A project in-between luv and luvit, lua packages."
+  homepage "https://github.com/luvit/luvi"
+  url "https://github.com/luvit/luvi/releases/download/v2.1.8/luvi-src-v2.1.8.tar.gz"
+  sha256 "1ef449c6a76bcf7a03c8bbcdd5423b0366d63b10c52fd463ca2ba8aaf9248781"
+
+  depends_on "cmake" => :build
+
+  def install
+    system "make", "regular-asm"
+    system "make"
+    bin.install "build/luvi"
+  end
+
+  test do
+    system "make", "test"
```

Comment:
It won't do anything. The `makefile` won't be moved across to the `testpath`, so theoretically it's likely the system will just exit with `make: *** No rule to make target'.` or similar. Once the installation has finished you have access to whatever was installed; the `buildpath` is gone.

If you can write an example to the testpath and use that for testing, that'd be awesome.